src/Transports.AspNetCore/GraphQLHttpMiddlewareOptions.cs

```diff
@@ -95,7 +97,8 @@ public class GraphQLHttpMiddlewareOptions : IAuthorizationOptions
     public GraphQLWebSocketOptions WebSockets { get; set; } = new();
 
     /// <summary>
-    /// The Content-Type to use for GraphQL responses
+    /// The Content-Type to use for GraphQL responses, if it matches the 'Accept'
+    /// HTTP request header. Defaults to "application/graphql-response+json; charset=utf-8".
     /// </summary>
-    public string ResponseContentType { get; set; } = GraphQLHttpMiddleware.CONTENTTYPE_GRAPHQLJSON;
+    public MediaTypeHeaderValueMs DefaultResponseContentType { get; set; } = MediaTypeHeaderValueMs.Parse(GraphQLHttpMiddleware.CONTENTTYPE_GRAPHQLRESPONSEJSON);
```

Comment:
I suggest to move CONTENTTYPE_GRAPHQLRESPONSEJSON into options class.

Reply:
If you wish to make a PR in this regard, please make it separately from other of your comments here - thanks!  As of now, I'm not sure why this would make sense to do, given that this and a number of other constants are required in the middleware class.  We should keep all of the content type / media type constants together.

Reply:
Nope. Now I see a bunch of other constants in GraphQLHttpMiddleware.